p2p/host/autorelay/autorelay_test.go

```diff
@@ -457,6 +461,40 @@ func TestIncorrectInit(t *testing.T) {
 	_ = newPrivateNode(t)
 }
 
+func expectDeltaInAddrUpdated(t *testing.T, addrUpdated event.Subscription, expectedDelta int) {
+	t.Helper()
+	delta := 0
+	for {
```

Comment:
Is it ok that adding and removing would cancel out here?

Reply:
I think so? I don't think it'll matter really